src/cascade/executor/argument_parser.py

```diff
@@ -208,7 +216,8 @@ def _logging_configure_mathlog(mvid, epiviz_log_dir):
             logging.warning(f"Could not write to math log at {log_file} even though "
                             f"directory {math_log_dir} exists: {mhe}")
             return
-        fmt = "%(levelname)s %(asctime)s %(funcName)s: %(message)s"
+        # The br is an HTML tag to add a line break.
+        fmt = "%(levelname)s %(asctime)s %(funcName)s: %(message)s<br />"
```

Comment:
This implies that the math log is _only_ to be shown in the GUI since having random HTML in it makes it less readable in other contexts. Be nice to have the viz people handle this instead but that's fine.